### PR TITLE
Update sp.json

### PR DESCRIPTION
**Pull request by oscampo**
Added "two letter" short names to some books

### CODE REVIEW

**Comment by oscampo** [comment]
Hi @tim-hub 
I added some short names on a common format on spanish ("two letter" short names), and tryed to merge, but I obtained an error, please could you check it ?

**Comment by tim-hub** [comment]
It complains the duplication, break the unique constraint

File: packages/bible-book-names-intl/src/data/translations/sp.json [hunk 1]
```diff
@@ -94,19 +94,19 @@
...
     "name": "Ester",
-    "shortNames": [
+    "shortNames": ["Est"
     ],
     "startNumber": 0
   },
   "18": {
     "name": "Job",
-    "shortNames": [
+    "shortNames": ["Job"
```
Comment (tim-hub) [inline packages/bible-book-names-intl/src/data/translations/sp.json:109]: here, break the unique constraint.

**Review by tim-hub — CHANGES_REQUESTED** [review]
test failed because of book Job

**Comment by oscampo** [comment]
> test failed because of book Job

ok.. fixed and updated

**Comment by tim-hub** [comment]
closed this, https://github.com/AntiochTechnology/bible-reference-toolkit/pull/12

**Pull request closed without merging**